incubator/elasticsearch-exporter/values.yaml

```diff
@@ -0,0 +1,71 @@
+## number of exporter instances
+##
+replicaCount: 1
+
+## restart policy for all containers
+##
+restartPolicy: Always
+
+image:
+  repository: justwatch/elasticsearch_exporter
+  tag: 1.0.2
+  pullPolicy: IfNotPresent
+
+resources:
```

Comment:
Defaults for resources should be commented out.

```yaml
resources: {}
  # requests:
  #   cpu: 100m
  #   memory: 128Mi
  # limits:
  #   cpu: 100m
  #   memory: 128Mi
```

Reply:
@unguiculus not sure if i can follow you here. if i comment the values out as requested by you, then there are no default values defined anymore. can you please explain?

Reply:
See https://github.com/unguiculus/charts/blob/master/REVIEW_GUIDELINES.md#configuration

Reply:
Thx for the clarification 👍

Reply:
I removed the default resources as suggested in the review guide lines.